mcs/class/corlib/win32_xammac_net_4_5_corlib.dll.sources

```diff
@@ -0,0 +1 @@
+#include win32_net_4_x_corlib.dll.sources
```

Comment:
What is the `win32_xammac_net_4_5_corlib.dll.sources` file for?

Reply:
Should go away